src/Cpt.php

```diff
@@ -356,9 +360,11 @@ public function set_supports( $support ) {
 	 */
 	private function update_arg( $name = '', $value = '' ) {
 		if ( ! empty( $name ) ) {
-			$this->set_args( array(
-				$name => $value,
-			));
+			$this->set_args(
+				array(
```

Comment:
@honekun can we use the array short syntax here?

Reply:
Updated array syntax.
thanks @mannuel